.buildkite/pipeline.yml

```diff
@@ -53,3 +58,18 @@ steps:
         allow_failure: true
       - step: "test-win"
         allow_failure: true
+
+  - label: "Publish docker image"
+    key: "publish"
+    command: ".buildkite/scripts/publish.sh"
+    agents:
+      provider: "gcp"
+    depends_on:
+      - step: "test-linux"
+        allow_failure: false
+      - step: "test-win"
+        allow_failure: false
+      - step: "build"
+        allow_failure: false
+      - step: "lint"
+        allow_failure: false
```

Comment:
Could you check if making a unit test fail this step is run in Jenkinsfile ? I think if there is a failure this step does not run in Jenkins...

Reply:
I see the latest commits, thanks for checking this out @leo-ri